### PR TITLE
0060_Permutation_Sequence.py

### PR DESCRIPTION
**Pull request by venkata-sreeram**
**IMPORTANT: Please do not create a Pull Request without an issue.**

### All Submissions:

* [X] I have read the CONTRIBUTING document.
* [X] My code is written in **Python 3** and is ending with _.py_
* [X] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [X] I have checked that my submission does pass the test on LeetCode.com 
* [X] Does your filename follow the naming Conventions?
* [X] Have you linked your PR to an Issue?

### CODE REVIEW

**Review by vJechsmayr — COMMENTED** [review]
Thanks for contributing! Hacktoberfest-Accepted (will be merged if first assignee to issue does not fix problems)

Runtime: 32 ms

Happy Hacktoberfest!

File: LeetCode/0060_Permutation_Sequence.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class Solution:
+    def getPermutation(self, n: int, k: int) -> str:
+        self.k = k
+    
+        def backtrack(path, remain):
```
Comment (Vulwsztyn) [inline LeetCode/0060_Permutation_Sequence.py:5]: you could add typings

File: LeetCode/0060_Permutation_Sequence.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+            if not remain:
+                self.k -= 1
+                if self.k == 0:
+                    return path
+                return ''
+                
+            for i in range(len(remain)):
+                if self.k > math.factorial(len(remain)-1):
```
Comment (Vulwsztyn) [inline LeetCode/0060_Permutation_Sequence.py:13]:
```suggestion
                if self.k > math.factorial(len(remain) - 1):
```

File: LeetCode/0060_Permutation_Sequence.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+                    self.k -= math.factorial(len(remain)-1)
```
Comment (Vulwsztyn) [inline LeetCode/0060_Permutation_Sequence.py:14]:
```suggestion
                    self.k -= math.factorial(len(remain) - 1)
```

Comment (Vulwsztyn) [reply LeetCode/0060_Permutation_Sequence.py:14]: `math.factorial(len(remain) - 1)` is counted twice you could assign it to a variable

File: LeetCode/0060_Permutation_Sequence.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+                    continue
+                ans = backtrack(path + remain[i], remain[:i]+remain[i+1:])
```
Comment (Vulwsztyn) [inline LeetCode/0060_Permutation_Sequence.py:16]:
```suggestion
                ans = backtrack(path + remain[i], remain[:i] + remain[i + 1:])
```

File: LeetCode/0060_Permutation_Sequence.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+                if ans: 
+                    return ans
+            return ''
+        nums = list(map(str, range(1, n+1)))
```
Comment (Vulwsztyn) [inline LeetCode/0060_Permutation_Sequence.py:20]:
```suggestion
        nums = list(map(str, range(1, n + 1)))
```

**Comment by vJechsmayr** [comment]
@venkata-sreeram No need to change that typo 

@Vulwsztyn Please accept different code styles - typo is no need to change

**Comment by Vulwsztyn** [comment]
These are mere suggestions and afaik they require no more work to be implemented than clicking the tick near them. Nevertheless I will refrain from suggesting changes that merely improve the code's readibility.

**Pull request closed without merging**